smart-contracts/contracts/dex-router-osmosis/src/tests/mod.rs

```diff
@@ -1,3 +1,4 @@
+mod initialize;
```

Comment:
Could we have the test-tube tests in a separate directory or with some tag such that we can run them independently of the unit-tests?

Reply:
i thing adding a tag makes sense as we have grouped test by the functions?